google-home-community.groovy

```diff
@@ -4355,15 +4306,19 @@ private deviceTypeTraitsFromSettings(deviceTypeName) {
 
 private deviceTypeFromSettings(deviceTypeName) {
     def deviceType = [
-        name:             deviceTypeName,
-        display:          settings."${deviceTypeName}.display",
-        type:             settings."${deviceTypeName}.type",
-        googleDeviceType: settings."${deviceTypeName}.googleDeviceType",
-        devices:          settings."${deviceTypeName}.devices",
-        traits:           deviceTypeTraitsFromSettings(deviceTypeName),
-        confirmCommands:  [],
-        secureCommands:   [],
-        pinCodes:         [],
+        name:                     deviceTypeName,
+        display:                  settings."${deviceTypeName}.display",
+        type:                     settings."${deviceTypeName}.type",
+        googleDeviceType:         settings."${deviceTypeName}.googleDeviceType",
+        devices:                  settings."${deviceTypeName}.devices",
+        traits:                   deviceTypeTraitsFromSettings(deviceTypeName),
+        confirmCommands:          [],
+        secureCommands:           [],
+        pinCodes:                 [],
+        useDevicePinCodes:        [],
+        pinCodeAttribute:         [],
+        pinCodeValue:             [],
+        returnUserIndexToDevice:  [],
```

Comment:
An empty list doesn't make much sense as the default value for these, perhaps something like this:
```suggestion
        useDevicePinCodes:        false,
        pinCodeAttribute:         null,
        pinCodeValue:             null,
        returnUserIndexToDevice:  false,
```

Reply:
Thanks, and done.